daml-foundations/daml-ghc/language-server/src/DA/Service/Daml/LanguageServer/CodeLens.hs

```diff
@@ -6,6 +6,7 @@
 -- | Gather code lenses like scenario execution for a DAML file.
 module DA.Service.Daml.LanguageServer.CodeLens
     ( handle
```

Comment:
Do we actually use this function somewhere outside of this module? If not, I think there are similar cases in other modules defining handlers.

Reply:
Yep, this is some of the dead code to clean up (in this case just an export, but still useful to kill).